Makefile

```diff
@@ -27,3 +28,8 @@ kube-update: update-manifests
 
 update-manifests:
 	sed 's#\(image:\) .*#\1 $(IMAGE)#' manifests/deis-logger-fluentd-daemon.yaml > manifests/deis-logger-fluentd-daemon.tmp.yaml
+
+build-plugin:
+
+	rake --rakefile=deis-output/Rakefile build
+	cp deis-output/pkg/*.gem rootfs/
```

Comment:
All of that happens within the fluentd image itself. No need to build this plugin outside of the image.